plugins/filter/wtor.py

```diff
@@ -12,6 +12,61 @@
 from __future__ import absolute_import, division, print_function
 
 __metaclass__ = type
+
+DOCUMENTATION = r"""
+name: filter_wtor_messages
+author: Demetrios Dimatos (@ddimatos)
+version_added: "1.2.0"
```

Comment:
should this be 1.9.0?

Reply:
Actually no, it came out in ibm_zos_core 1.2.0 , documentation of filters was never a warning until ansible 2.16 (sidecar) was developed and ansible-lint became part of certification, so 1.2.0 is correct. Once we drop 2.14 all this will need to be using sidecare (YAML) over stub python files.